### DISCUSSION TITLE
How to pick which channel to watch?

### DISCUSSION THREAD

**Discussion by sussum2432** [iptv-org/community#504] [Q&A] [answered]
Hi
I'm first time users of this IP TV
I put the link in the VLC medias player
It starts from first channel by alphabet (some ukrainian). How do i see list of all channels and skip to the one i want?

**Comment by ThetaX55** [comment]
Find some .m3u . is a bunch of m3u8 in it or a bunch of channels. 
Maybe on this board or whatever you find. 
m3u8 should be a single channel.

**Comment by Dum4G** [comment, marked as answer]
Click this button to see your playlist
![image](https://github.com/iptv-org/iptv-org.github.io/assets/5430116/813d4efb-558c-4f09-b030-1a24ed94c4da)
